### ISSUE TITLE
How to just display start the remotely installed VSCode ?

### ISSUE THREAD

**Issue by cdevr** [cdr/sshcode#164]
So that you can use manual extensions on the remote host

**Comment by Merith-TK** [comment]
So to recap from what i am getting from your issue

you want to use the extensions on the REMOTE, and not sync the ones on the Client?

I mean this in theory would be easyish to implement, just adding a flag, 

Side nore, for future issues, i would reccomend explaining how you would want it to work so that we can understand more

The title of a issue is a summary, 

And SSHCODE works by using ssh to run commands on the REMOTE, which it downloads a custom version of cdr/code-server and runs it, before Proxying the port that code-server is running on, back to local host.

In theory it would be possible to use the REMOTES local extensions/settings,

**Comment by cdevr** [comment]
That's correct. The issue is that some organisations have private plugins for internal systems that aren't installed, or allowed to be installed on laptops (nor would they work: they depend on software only installed on the desktops). It would be great to be able to use them through this tool.

**Comment by Merith-TK** [comment]
Well, Most of your private plugins should work just fine if you make it use vscode-datadir
Specific Microsoft plugins such as the Remote toolset will not work at all, as they are hardcoded to only work with official microsoft releases, 

If the private plugins are not hardcoded to only work with a specific setup, then yeah they should work, i would recommend forking this and adding the feature as a flag for you,

Currently im waiting on my fork to be added into the repo to make the tool work on windows.

**Comment by Merith-TK** [comment]
A workaround you could do to make it work, is to make symlinks from the VSCode data dir to the sshcode-server datadir,

**Comment by cdevr** [comment]
How do I find the vscode datadir ? I was trying just that earlier today and
couldn't find it.

**Comment by cdevr** [comment]
I mean of course the one for the vscode-server that this tool is using.

**Comment by Merith-TK** [comment]
Look iver the source code, should be the remoteExtensionsDir variable in
sshcode.go
